.github/workflows/check-archs-consistency.yml

```diff
@@ -30,4 +30,4 @@ jobs:
       - name: Install node dependencies
         run: yarn
       - name: Check Android Paper & Fabric generated interfaces consistency
-        run: yarn architectures-consistency-check
+        run: yarn architectures:consistency-check
```

Comment:
```suggestion
        run: yarn architectures-consistency-check
```

iirc

Reply:
Yep, I've missed that. However, those CI were not run on this PR and I've already found it and changed in #3515

Reply:
Well, I thought about it for a while and I've decided to fix that here 😅 Done in [cd0f3ed](https://github.com/software-mansion/react-native-gesture-handler/pull/3510/commits/cd0f3edf776979f9fde707ef4ee86f745f39b2cf)